packages/@storefront/flux-capacitor/src/core/actions/validators.ts

```diff
@@ -181,3 +181,40 @@ export const isNotFetching: Validator<boolean> = {
     !Selectors.infiniteScroll(state).isFetchingBackward,
   msg: 'is already fetching'
 };
+
+export const hasValidLabels: Validator<Actions.Payload.Sort> = {
+  func: ({ labels }) => {
+    return !labels
+      || (
+        Array.isArray(labels)
+        && !!labels.length
+        && labels.every((label) => typeof label === 'string')
+      );
+  },
+  msg: 'if present, labels must be an array of strings',
+};
+
+export const hasValidOptions: Validator<Actions.Payload.Sort> = {
+  func: ({ options }) => {
+    return (
+      Array.isArray(options)
+      && !!options.length
+      && options.every((option) => typeof option.field === 'string'
```

Comment:
If you have to break the conditional into multiple lines, I'd put the first condition on its own line as well.

Reply:
Noted, will apply.